STATUS: INLINE SUCCESSFUL
MODIFIED: com.j256.ormlite.logger.LoggerFactoryTest.testSetLogFactory
MODIFIED FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/logger/LoggerFactoryTest.java
CLEANER: com.j256.ormlite.logger.LoggerFactoryTest.testLogFactoryProperty
CLEANER FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/logger/LoggerFactoryTest.java
POLLUTER: com.j256.ormlite.logger.LoggerFactoryTest.testSetLogFactory
POLLUTER FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/logger/LoggerFactoryTest.java
ORIGINAL CLEANER SIZE: 5
NEW CLEANER SIZE: 1
ELAPSED TIME: 20.197999954223633
==========================
@@ -93,0 +93,3 @@
+ {
+     LoggerFactory.setLogBackendFactory(null);
+ }
